### PR TITLE
Create fa.js

### PR DESCRIPTION
**Pull request by saadeghi**

### CODE REVIEW

File: fa.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"%d unread message": {
+		"one": "%d پیام جدید",
+		"other": "%d پیام جدید"
```
Comment (MadLittleMods) [inline fa.js]: The `one` and `other` variations seem to be the same `پیام جدید`. I am completely unfamiliar with Persian (Farsi) but should there be a difference for plurals?

**Comment by MadLittleMods** [comment]
Hey @saadeghi,

Thanks for the translation. All I can do is run it through Google translate and they seem to be fine. Only thing that caught my eye is "unread" translating to "new". Just want to make sure there isn't a separate word, etc.

**Comment by saadeghi** [comment]
@MadLittleMods Thanks for checking, my bad :) I fixed it.

**Comment by MadLittleMods** [comment]
Thanks for the contribution @saadeghi :grinning:  :+1:

As soon as we update this dependency in the webapp and make a release, you will start seeing the translation.

**Pull request merged**